packages/ui-app/src/LabelHelp.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017-2019 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BareProps } from './types';
+
+import React from 'react';
+import styled from 'styled-components';
+
+import Icon from './Icon';
+import { classes } from './util';
+import Tooltip from './Tooltip';
+
+type Props = BareProps & {
+  help: React.ReactNode
+};
+
+type State = {
+  tooltipOpen: boolean
+};
+
+const Wrapper = styled.div`
+  cursor: help;
+  display: inline-block;
+  line-height: 1rem;
+  margin: 0 0 0 0.25rem;
+`;
+
+export default class Labelled extends React.PureComponent<Props, State> {
```

Comment:
Should be `LabbelHelp` I guess. I wonder how this can work without warning as it.

Reply:
Ahh, doesn't matter what the class is named for use. So we do

`export { default as LabelHelp } from './LabelHelp'` - so it is actually called "default" is export.

But yes, so it works whatever it is called, but should be changed. Copy error there.